### ISSUE TITLE
sudo make install fails

### ISSUE THREAD

**Issue by mrx23dot** [autotrace/autotrace#114]
```
  ./autogen.sh
  ./configure
  make
  make check
  ./autotrace
     //works locally
  sudo make install 

cd ..
autotrace
/usr/bin/autotrace: error: '/usr/bin/.libs/autotrace' does not exist
This script is just a wrapper for autotrace.
See the libtool documentation for more information.

/usr/local/bin/autotrace
/usr/local/bin/autotrace: error while loading shared libraries: libautotrace.so.3: cannot open shared object file: No such file or directory

```

**Comment by lemenkov** [comment]
Cannot confirm that. It installs quite fine on my Fedora. Maybe your system doesn't have /usr/local/lib(64) as a LDPATH or LD_LIBRARY_PATH?

Could you please post output of `ldd /usr/local/bin/autotrace` ?

**Comment by lemenkov** [comment]
@jnweiger thank you! I think we can close this ticket.

**Comment by lemenkov** [comment]
Should be addressed in 36a72de915066fb8b9e74d7699d103e20971e241.
